bin/www

```diff
@@ -80,6 +81,8 @@ function onListening() {
  * Listen on provided port, on all network interfaces.
  */
 initDB().then(() => {
+  fixLabelCounts();
+  setInterval(fixLabelCounts, MAX_LEVEL_TIME_SEC * 1000);
```

Comment:
I wonder if we should not run this automatically to start off; maybe we can run it manually a few times first to make sure it doesn't do something unexpected?

Reply:
Hmm, I would think if there were problems they would show up on the first run. Now that there are tests passing I feel more confident about it. But if you want to try running it before merging go ahead.

Reply:
ok, sounds good to me. We can try running another 100 rounds on mturk today